compiler/tests-full/stdlib.cma.expected.js

```diff
@@ -586,10 +585,9 @@
             (oc);
     /*<<stdlib.ml:390:49>>*/ }
    function close_out_noerr(oc){
-     /*<<stdlib.ml:392:2>>*/ try{
-      /*<<stdlib.ml:392:7>>*/  /*<<stdlib.ml:392:7>>*/ caml_ml_flush(oc);
-    }
-    catch(_u_){}
+     /*<<stdlib.ml:392:2>>*/ b:
+    try{ /*<<stdlib.ml:392:7>>*/  /*<<stdlib.ml:392:7>>*/ caml_ml_flush(oc);}
+    catch(_u_){break b;}
```

Comment:
This looks unnecessary.

Reply:
Yes, probably. I don't understand why we don't have the same issue with the current implementation.

Reply:
In the current implementation, we have the following piece of code
```ocaml
    (* Create an artificial frontier when we pop an exception handler *)
    let rec keep_front pc =
      if Hashtbl.find preds pc > 1
      then (* already part of a merge node *) false
      else
        match Addr.Map.find pc blocks with
        | { body = []; branch = Return _, _; _ } -> false
        | { body = []; branch = Stop, _; _ } -> false
        | { body = []; branch = Branch (pc', _), _; _ } -> keep_front pc'
        | _ -> true
    in
```
Which drops the poptrap frontier if there is a sequence of empty blocks ending in a merge node.

Reply:
So, I suppose we can do the same?

Reply:
Yes, I've pushed a change

Reply:
Can this be used in wasm_of_ocaml ?